WordPress/Classes/Services/ReaderSiteService.h

```diff
@@ -1,6 +1,9 @@
 #import <Foundation/Foundation.h>
 #import "ReaderTopicService.h"
-#import "CoreDataService.h"
+#import "CoreDataStack.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wnullability-completeness"
```

Comment:
Is there a way to remove it and add all the required nullability modifiers? It looks like it's just the new property that needs `nonnull`.

Reply:
I'll follow up on this...

Reply:
https://github.com/wordpress-mobile/WordPress-iOS/pull/24373